### ISSUE TITLE
During startup `Host name lookup failure : Resource temporarily unavailable`

### ISSUE THREAD

**Issue by paulmenzel** [NLnetLabs/unbound#263]
Using Unbound 1.9.1 and using the systemd unit below to start it

```
$ systemctl cat unbound
# /etc/systemd/system/unbound.service
[Unit]
After=network.service gateway.service

[Service]
Type=forking
ExecStartPre=-/usr/sbin/unbound-startup setup-var
ExecStart=/usr/sbin/unbound
PIDFile=/var/unbound/unbound.pid

[Install]
WantedBy=network.target
```

we experience host name lookup failures of other programs, ordered after Unbound, on startup.

For example, the script

    reply="$(netcat -w 1 afk 236 </dev/null)"

returns the error below.

    afk: forward host lookup failed: Host name lookup failure : Resource temporarily unavailable

Due to this, we are also experience problems with NFS.

We are using DNSSEC, which might slow things down for Unbound in the beginning.

Can this error be avoided?

**Comment by gthess** [comment]
This seems to be related to the recently closed #296 
Can you try the following in your systemd unit file and see if it solves the issue?
```
After=network-online.target
Before=nss-lookup.target
Wants=network-online.target nss-lookup.target
```

**Comment by gthess** [comment]
Closing this because of inactivity. Feel free to ping me here if you want this reopened.
